scripts/db/src/get-user.ts

```diff
@@ -75,15 +75,73 @@ async function getByEmail(email: string, callback: DbScriptCallback) {
     `
     const result = await client.query<PersonResult>(query, [email])
 
-    /** Close the connection */
-    await client.end()
-
+    // If there are no rows returned, get out of here
     if (result.rows.length === 0) {
+      // Close the connection before returning
+      await client.end()
```

Comment:
I don't know the implications of failing to close the client, but possibly this should be inside a `finally`